app/tests/utils.py

```diff
@@ -20,23 +29,22 @@ def decompose_mobile_invisible_elements(soup):
         invisible_element.decompose()
 
 
-def evaluate_table_body_rows(soup, expected_results):
+def get_table_rows_cell_values(table):
     """
-    Gets all rows and cell values for a table inside a page after decomposing invisible items then asserts the result
-    :param soup: BeautifulSoup
-    :param espected_results: string[][]
+    Returns all rows and cell values for a table that has been found using BeautifulSoup
+    :param table: BeautifulSoup object
+    Returns:
+    string[][], e.g. [["row_1_value_1", "row_1_value_2"], ["row_2_value_1", "row_2_value_2"]]
```

Comment:
not familiar with this return type syntax, is it for python? I am familiar with the mypy List[List[str]]

Reply:
im used to typescript to this is why I wrote it like this, but yes its List[List[str]]

Reply:
nitpic: could we omit or change to the python syntax

Reply:
@anthonyhashemi yes should be changed now